src/naksu/mebroutines/install/install.go

```diff
@@ -15,15 +15,28 @@ import (
 )
 
 // GetServer downloads vagrantfile and starts server
+// nolint: gocyclo 
 func GetServer(newVagrantfilePath string) {
 	// Create ~/ktp if missing
 	progress.TranslateAndSetMessage("Creating ~/ktp")
-	var ktpPath = createKtpDir()
+	ktpPath, errKtpPath := createKtpDir()
+
+	if errKtpPath != nil {
+		mebroutines.ShowErrorMessage(fmt.Sprintf(xlate.Get("Could not create ~/ktp to %s"), ktpPath))
+		return
+	}
+
 	log.Debug(fmt.Sprintf("ktpPath is %s", ktpPath))
 
 	// Create ~/ktp-jako if missing
 	progress.TranslateAndSetMessage("Creating ~/ktp-jako")
-	var ktpJakoPath = createKtpJakoDir()
+	ktpJakoPath, errKtpJakoDir := createKtpJakoDir()
+
+	if errKtpJakoDir != nil {
+		mebroutines.ShowErrorMessage(fmt.Sprintf(xlate.Get("Could not create ~/ktp-jako to %s"), ktpJakoPath))
+		return
+	}
+
```

Comment:
Esim. nämä hakemistojen luonnit voisi refaktoroida yhdeksi apufunktioksi, vaikka `ensureNaksuDirectoriesExist` tms.

Reply:
Päädyin nyt tekemään näin, vaikka minusta on höperöä tehdä asioita vain sen vuoksi, että joitain rakenteita funktiossa on 11 sijaan 10. Oikea ratkaisu olisi paketoida tämä virtuaalikoneen hallinta omaksi paketikseen, mutta nyt ei ole sen aika.